### PR TITLE
Fix ocean file locations in archive job

### LINKED ISSUE #1709: wrong directory name for ocean in archiving step
**Expected behavior**
in coupled runs, ocean 2D, 3D and daily files should be archived to HPSS

**Current behavior**
they are not being saved to HPSS. 
**Machines affected**

**To Reproduce**
run current g-w in coupled mode

**Context**

**Detailed Description**
This is due to the wrong directory name in "hpssarch_gen.sh" script. See https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/hpssarch_gen.sh#L262-L265
 I think this is because g-w had a major directories re-structure work couple of months ago. For ocean files they are catagoried as 

- model/ocean/history (where daily files are there)
- products/ocean/2D,  3D  (where 2D and 3D files are there)
but no corresponding modification were made in hpssarch_gen.sh

**Additional Information**

**Possible Implementation**

### PR DESCRIPTION
**Pull request by WalterKolczynski-NOAA**
**Description**

During the COM refactor, a location for the ocn/ice daily files was defined within the products directory. However, the daily files are output directly by the model and land in the history directory. The archive job was looking in the otherwise unused daily product directory for the files and not finding them.

The 2D and 3D ocean files had the opposite problem: archive was looking in the history directory, but these files have their own directories in products.

This update now:
- Removes the COM variable for the daily files
- Updates the archive job to:
  - pull the daily ocean files from history
  - pull 2D ocean files from the 2D product directory
  - pull 3D ocean files from the 3D product directory
 
Fixes #1709

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested?**

- [x] Forecast-only S2S on Orion with local archiving
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes

### CODE REVIEW

**Comment by JessicaMeixner-NOAA** [comment]
Thanks @WalterKolczynski-NOAA !

File: ush/hpssarch_gen.sh [hunk 1]
```diff
@@ -259,10 +259,10 @@ if [[ ${type} = "gfs" ]]; then
     touch ocn_xsect.txt
     touch ocn_daily.txt
     echo "${COM_OCEAN_INPUT/${ROTDIR}\//}/MOM_input" >> ocn_2D.txt
-    echo "${COM_OCEAN_HISTORY/${ROTDIR}\//}/ocn_2D*" >> ocn_2D.txt
-    echo "${COM_OCEAN_HISTORY/${ROTDIR}\//}/ocn_3D*" >> ocn_3D.txt
+    echo "${COM_OCEAN_2D/${ROTDIR}\//}/ocn_2D*" >> ocn_2D.txt
```
Comment (aerorahul) [inline ush/hpssarch_gen.sh:262]: I think you can remove the `ocn_daily.txt` file being touched on line 260.

**Review by aerorahul — APPROVED** [review]
Looks good.
@jiandewang please verify the contents of the hpss archive to ensure the required files are present.

**Comment by jiandewang** [comment]
will make an end-to-end run to verify the fixing

**Comment by jiandewang** [comment]
my end-to-end run finished, archiving step works as expected now.

**Review by jiandewang — APPROVED** [review]

**Pull request merged**